### PR TITLE
Remove null origin issue

### PR DESCRIPTION
**Pull request by devd**
See https://lists.w3.org/Archives/Public/public-webappsec/2017May/0022.html

***
[Preview](https://s3.amazonaws.com/pr-preview/devd/fetch/patch-1.html) | [Diff](https://s3.amazonaws.com/pr-preview/whatwg/fetch/92228c4...devd:ebcdd85.html)

### CODE REVIEW

File: fetch.bs [hunk 1]
```diff
@@ -4082,10 +4082,6 @@ Entries may be removed before that moment arrives.
  <li>
   <p>If <var>credentials</var> is `<code>true</code>`, return success.
```
Comment (annevk) [inline fetch.bs]: This `<li>\n  <p>` should become `<li><p>` since the `<li>` no longer contains multiple `<p>`s.

**Comment by annevk** [comment]
I checked and surprisingly there's no corresponding issue for this.

**Comment by annevk** [comment]
You should also add your name to the acknowledgments section (unless you don't want to be listed).

**Comment by devd** [comment]
done

**Comment by annevk** [comment]
Thanks!

**Pull request merged**